src/FixedDataTableColumnResizeHandle.js

```diff
@@ -141,9 +141,15 @@ var FixedDataTableColumnResizeHandle = createReactClass({
   },
 
   _onMove(/*number*/ deltaX) {
+
+    if (deltaX === 0) {
+      return;
```

Comment:
It basically stopped the resize callbacks when they were essentially noops. So it doesn't really do anything but I think it makes for a cleaner api.